plugins/by-name/kitty-scrollback/default.nix

```diff
@@ -0,0 +1,7 @@
+{ lib, ... }:
+lib.nixvim.plugins.mkNeovimPlugin {
+  name = "kitty-scrollback";
+  package = "kitty-scrollback-nvim";
+
+  maintainers = [ lib.maintainers.nim65s ];
```

Comment:
Should we have a `settingsExample` inspired by something in https://github.com/mikesmithgh/kitty-scrollback.nvim#plugin-configuration ?